uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-action-dropdown.html

```diff
@@ -0,0 +1,20 @@
+<div uib-dropdown dropdown-append-to-body ng-if="vm.actions.length">
+  <a href class="btn btn-default btn-inline" uib-dropdown-toggle>
+    <i class="fa fa-ellipsis-v"></i>
+  </a>
+  <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu>
+    <li ng-repeat="action in vm.actions"
+      ng-switch="['edit', 'respond', 'view'].indexOf(action) > -1">
```

Comment:
Done. Though I am not quite sure we want to use `openPopup` in the controller name since this defines a view and not business logic. I called it `isDirectAction`.